mars/web/api.py

```diff
@@ -175,6 +182,40 @@ def get(self):
         self.write(json.dumps(workers_num))
 
 
+class MutableTensorHandler(MarsApiRequestHandler):
+    def get(self, session_id, name):
+        try:
+            meta = self.web_api.get_mutable_tensor(session_id, name)
+            self.write(json.dumps(meta))
+        except:  # noqa: E722
+            self._dump_exception(sys.exc_info())
+
+    def post(self, session_id, name):
+        try:
+            # If the request contains no body payload, it is seal, otherwise it is create
```

Comment:
I still think it is better to use a customized header or a query string argument to define the action of ``POST``.

Reply:
Fixed. A parameter `action` has been added.